packages/elements/src/components/file/file.scss

```diff
@@ -31,8 +31,8 @@ $mutant-squiggly: 'Survived', 'NoCoverage';
   }
 }
 
-svg.mutant-dot.selected {
-  @apply fill-sky-400;
+polygon.mutant-dot-polygon {
+  @apply stroke-gray-800;
```

Comment:
For single tailwind classes I think it makes more sense to inline it. I.e. instead of the `polygon.mutant-dot-polygon` selector, add the `stroke-gray-800` class to the polygon directly. Then this bit can be removed